lib/PluginBase.js

```diff
@@ -61,66 +54,66 @@ class PluginBase {
     /**
      * Get a State from State DB
      *
-     * @param id {string} id of state to retrieve
-     * @param callback {function} function to be called with the result. Signature for callback is (err, state)
+     * @param {string} id id of the state to retrieve
```

Comment:
The order of `{type} paramName` was wrong everywhere.

Reply:
uups ... sorry